array_analyzer/workflows/registration_workflow.py

```diff
@@ -144,13 +144,13 @@ def point_registration(input_folder, output_folder, debug=False):
         )
 
         # Estimate and remove background
+        im_crop = im_crop.astype('float64')
+        im_crop /= 255.0
         background = img_processing.get_background(
             im_crop,
             fit_order=2,
-            normalize=True,
+            normalize=False,
```

Comment:
False is default so you can remove that parameter when working with normalized images.